weio/tests/test_bladed.py

```diff
@@ -0,0 +1,25 @@
+import os
+import numpy as np
+import re
+import pandas as pd
+import weio
+import unittest
+from helpers_for_test import MyDir, reading_test 
```

Comment:
it should be:
`from . helpers_for_test import ...`

Reply:
Your version works when you call the test file from the current directory, but it doesn't work when the test is called from another directory. It's a bit annoying unfortunately.

Reply:
`test_bladed.py` still does not work for me :-(